estate/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields
+
+class Property(models.Model):
```

Comment:
```suggestion


class Property(models.Model):
```
in python you should always put 2 lines above a class definition